src/Sylius/Behat/Context/Api/Shop/CheckoutContext.php

```diff
@@ -30,6 +30,8 @@
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Webmozart\Assert\Assert;
+use function Symfony\Component\String\u;
+
```

Comment:
```suggestion
```

Reply:
I need to use this function to change strings into camelCase ( im searching for it in "foreach")

Reply:
moved to StringInflector